enterprise/installation.md

```diff
@@ -0,0 +1,120 @@
+
+![SSL Setup](/Resources/public/img/docs/enterprise/02-ssl-setup.png)
+
+On the next screen you can upload the supplied .rli license key file.
+
+![Upload License](/Resources/public/img/docs/enterprise/03-upload-license.png)
+
+Once validated, you will be asked to set a password for the management console or to protect it with a different authentication mechanism.
+
+![Security Console](/Resources/public/img/docs/enterprise/04-secure-console.png)
+
+### Server Settings
+In the management console you will have to configure a few options before you can start the Private Packagist Enterprise application.
+
+#### E-mail
+Select options for your SMTP server or use the GMail option for a simplified form if want to send email through GMail. This step is required to allow Private Packagist to send transactional email to users.
+
+![Email Settings](/Resources/public/img/docs/enterprise/05-email-settings.png)
+
+## Application Startup
+Save and apply the settings to start Private Packagist Enterprise. Leave the Setup mode enabled.
+
+![Start Application](/Resources/public/img/docs/enterprise/06-save-start.png)
+
+Once the application has started, click on the open link to access the Private Packagist Enterprise Web Interface.
+
+![Open Application](/Resources/public/img/docs/enterprise/07-started-open.png)
+
+## Configuring an HTTP Proxy
+If accessing websites outside of your network, e.g. packagist.org, requires an HTTP proxy then you can
+enable and configure this in the Console Settings under the HTTP Proxy section which you can access in
+the Replicated management console by clicking on the gear icon in the menu bar.
+
+![HTTP Proxy](/Resources/public/img/docs/enterprise/07-01-http-proxy.png)
+
+In case the HTTP proxy should not be used to access certain domains or IPs then you can configure such
+a NO_PROXY list in the Private Packagist application in the Admin Panel under Global Configuration.
+
+![No HTTP Proxy](/Resources/public/img/docs/enterprise/07-02-http-proxy-no-proxy.png)
+
+## Authentication Setup
+Within Private Packagist Enterprise you now need to set up at least one integration with a third party service for user authentication. You can either use your on-premises version of GitHub, Bitbucket, or GitLab, or their public services on github.com, bitbucket.org or gitlab.com. Follow the instructions to create the respective required identifiers, tokens and secrets.
+
+* [GitHub (Enterprise) Integration Setup](./github-integration-setup.md)
+* [Bitbucket.org Integration Setup](./bitbucket-integration-setup.md)
+* [Bitbucket Server (Stash) Integration Setup](./bitbucket-server-integration-setup.md)
+* [GitLab Integration Setup](./gitlab-integration-setup.md)
+
+If your source code is hosted on another platform or you do not with to set up an integration then you can also enable [authentication with email and password](./authentication-email-password-setup.md).
```

Comment:
"wish" not "with".
Can you make this a list item, and make it clearer than you can setup multiple of these but need at least one?

Reply:
Updated. Right before the list it already says that you need at least one integration.